src/ChunkDef.h

```diff
@@ -112,7 +132,7 @@ class cChunkDef
 	/** Converts relative block coordinates into absolute coordinates with a known chunk location */
 	inline static Vector3i RelativeToAbsolute(const Vector3i & a_RelBlockPosition, int a_ChunkX, int a_ChunkZ)
 	{
-		return Vector3i(a_RelBlockPosition.x + a_ChunkX * Width, a_RelBlockPosition.y, a_RelBlockPosition.z + a_ChunkZ * Width);
+		return {a_RelBlockPosition.x + a_ChunkX * Width, a_RelBlockPosition.y, a_RelBlockPosition.z + a_ChunkZ * Width};
```

Comment:
That occurred to me, apicheck isn't complaining though. In both scenarios tolua will grab a pointer to the object, but for the pass-by-value it will dereference the pointer as soon as it gets it (so there's an extra copy there, could be important for tight loops, but plugins have so much overhead anyway). (and in pass-by-reference it's a const pointer, so there's no chance of accidentally changing the data in lua)